pkgs/by-name/bi/bic/package.nix

```diff
@@ -8,41 +8,46 @@
   gmp,
   flex,
   bison,
+  libffi,
+  pkg-config,
 }:
 
 stdenv.mkDerivation rec {
   pname = "bic";
-  version = "1.0.0";
+  version = "1.0.0-unstable-2022-02-16";
 
   src = fetchFromGitHub {
     owner = "hexagonal-sun";
     repo = "bic";
-    rev = "v${version}";
-    sha256 = "1ws46h1ngzk14dspmsggj9535yl04v9wh8v4gb234n34rdkdsyyw";
+    rev = "b224d2776fdfe84d02eb96a21880a9e4ceeb3065";
+    sha256 = "sha256-6na7/kCXhHN7utbvXvTWr3QG4YhDww9AkilyKf71HlM=";
```

Comment:
I think this is a good time to change `sha256` -> `hash`, although non-blocking

Reply:
I thought I should have changed it :D